packages/modeling/src/operations/extrusions/extrudeFromSlices.js

```diff
@@ -44,7 +44,7 @@ const extrudeFromSlices = (options, base) => {
     isCapped: true,
     callback: defaultCallback
   }
-  let { numberOfSlices, isCapped, callback } = Object.assign({ }, defaults, options)
+  const { numberOfSlices, isCapped, callback: generate } = Object.assign({ }, defaults, options)
```

Comment:
Why the code change? Did this pass tests?

Reply:
It did pass test. Nothing changed to the callee just a local variable name change. In standard js `callback` and `cb` expect an error argument see [here](https://eslint.org/docs/rules/handle-callback-err).

Reply:
That link seems to be rules in regards to proper error handling in callback functions, i.e. promises.

Was eslint throwing warnings for the 'callback' attribrute name?

Reply:
It's for node style callbacks, nothing to do with Promises.

Yes, if it's called `callback` or `cb` it is assumed to be a node style callback.

Reply:
I see... OK. I'm fine with this change then.